apps/admin/frontend/src/components/import_err_file_modal.tsx

```diff
@@ -26,6 +29,20 @@ const Content = styled.div`
   overflow: hidden;
 `;
 
+function errorCodeToMessage(
+  errorCode: ImportElectionResultsReportingError
+): string {
+  switch (errorCode.type) {
+    case 'parsing-failed':
+      return 'The results file could not be parsed.';
+    case 'conversion-failed':
+      return 'The contents of the file could not be converted.';
+    /* istanbul ignore next - compile time check */
+    default:
+      return 'An error occurred.';
```

Comment:
should we have `throwIllegalValue(errorCode)` here?

Reply:
[a6cc1c0](https://github.com/votingworks/vxsuite/pull/5344/commits/a6cc1c0da44cf8db044956ea99b016705c37d632)